ddtrace/opentelemetry/_span.py

```diff
@@ -77,9 +73,9 @@ def __init__(
 
         self._ddspan = datadog_span
         if record_exception is not None:
-            self._record_exception = record_exception
+            self._record_exception = record_exception if record_exception is not None else True
```

Comment:
@mabdinur 

Do these properties exist on `OtelSpan`?

Are we supposed to be using these private attributes?

Reply:
Yes these fields should be private. The OTEL API does not define any fields for the [Span](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/src/opentelemetry/trace/span.py#L57) object. These fields are defined in the [span implementation](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L774) in the OTEL SDK.

Reply:
In the Datadog API error messages and statuses are always recorded on spans. In the OTEL API this functionality is configurable.